embedchain/config/apps/OpenSourceAppConfig.py

```diff
@@ -26,6 +27,8 @@ def __init__(self, log_level=None, host=None, port=None, id=None, model=None):
             host=host,
             port=port,
             id=id,
+            db_type=db_type,
+            vector_dim=384,  # vector length created by embedding fn
```

Comment:
refer from a variable rather than the value.

Reply:
update: referring from VectorDimensions under providers